src/pages/ScholarshipForm.jsx

```diff
@@ -38,9 +41,21 @@ function ScholarshipForm() {
         {requiredTextField('name', 'Scholarship Name')}
         {requiredTextField('description', 'Description')}
         {requiredTextField('website', 'Website')}
-        <Button variant="contained" color="primary">
-          Submit
-        </Button>
+        <EnumSelectField
+          id="scholarshipType"
+          label="Scholarship Type"
+          value={formFieldStates.scholarshipType}
+          onChange={updateFn('scholarshipType')}
+          enums={ScholarshipType}
+        />
+        <div>
```

Comment:
Instead of a div I suggest looking into surrounding the form elements in a [Grid](https://material-ui.com/components/grid/) and setting the `spacing`

Reply:
Or alternatively, remove the `<div>` here and set `display="block"` for the button

Reply:
Or rather for the element above the button

Reply:
Or I guess a div works too 🤷